docs/integrations/graphite.md

```diff
@@ -0,0 +1,114 @@
+# Graphite
+
+This document is a getting started guide to integrating the M3 stack with Graphite.
+
+## Overview
+
+M3 supports ingesting Graphite metrics using the [Carbon plaintext protocol](https://graphite.readthedocs.io/en/latest/feeding-carbon.html#the-plaintext-protocol). We also support a variety of aggregation and storage policies for the ingestion pathway (similar to [storage-schemas.conf](https://graphite.readthedocs.io/en/latest/config-carbon.html#storage-schemas-conf)) that are documented below. Finally, on the query side, we support the full suite of [graphite query functions](https://graphite.readthedocs.io/en/latest/functions.html).
```

Comment:
nit: Change to "similar to storage-schemas.config when using Carbon"?

Reply:
super nit: `full suite` to `full set`

Reply:
Also... this is annoying but we should checkout which "version" is fully supported.  I don't think we're up to date with all the functions from the latest release, i.e. 1.1.x.